datacatalog/cloud-client/src/test/java/com/example/datacatalog/CreateFilesetEntryTests.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.datacatalog;
+
+import static org.junit.Assert.assertThat;
+
+import com.google.cloud.datacatalog.EntryGroupName;
+import com.google.cloud.datacatalog.EntryName;
+import com.google.cloud.datacatalog.v1beta1.DataCatalogClient;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.UUID;
+import org.hamcrest.CoreMatchers;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration (system) tests for {@link CreateFilesetEntry}. */
+@RunWith(JUnit4.class)
+public class CreateFilesetEntryTests {
+
+  private ByteArrayOutputStream bout;
+
+  private static String ENTRY_GROUP_ID =
+      "fileset_entry_group_" + UUID.randomUUID().toString().substring(0, 8);
+  private static String ENTRY_ID =
+      "fileset_entry_id_" + UUID.randomUUID().toString().substring(0, 8);
+  private static String LOCATION = "us-central1";
+  private static String PROJECT_ID = System.getenv().get("GOOGLE_CLOUD_PROJECT");
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(bout));
+    CreateEntryGroup.createEntryGroup(PROJECT_ID, ENTRY_GROUP_ID);
+  }
+
+  @After
+  public void tearDown() {
+    System.setOut(null);
+    bout.reset();
+
+    try (DataCatalogClient dataCatalogClient = DataCatalogClient.create()) {
+      dataCatalogClient.deleteEntry(
+          EntryName.of(PROJECT_ID, LOCATION, ENTRY_GROUP_ID, ENTRY_ID).toString());
+      dataCatalogClient.deleteEntryGroup(
+          EntryGroupName.of(PROJECT_ID, LOCATION, ENTRY_GROUP_ID).toString());
+    } catch (Exception e) {
+      System.out.println("Error in cleaning up test data:\n" + e.toString());
+    }
+  }
+
+  @Test
+  public void testCreateFilesetEntry() {
+    CreateFilesetEntry.createEntry(PROJECT_ID, ENTRY_GROUP_ID, ENTRY_ID);
```

Comment:
What I meant before was to merge these tests in with `LoopupEntryTests` (You can rename it something like `EntryTests` or `DataCatalog` tests if that's what you are concerned about). It's fine if you really prefer them to be in a different file since we do that sometimes, but typically it's only to break up a large number of tests into more manageable pieces. I'll leave it up to you if you want to break it up here.

I would prefer you make a separate test for `createEntry`. The first reason is it will make it easier to diagnose at a glance which snippet is causing the problem, and second there is work being done to create a dashboard of how the snippets are passing tests (but it requires each snippet to have it's own test).

Reply:
I see, it makes sense, I will work on it.

Reply:
done